tests/fixtures/ecma-features/impliedStrict/inner-strict-violation.src.js

```diff
@@ -0,0 +1,3 @@
+function inner() {
+    var private;
```

Comment:
This should throw a parsing error, correct? If so, the result file should have an error object, not an AST.